app/helpers/application_helper.rb

```diff
@@ -4,9 +4,10 @@ def format_datetime(datetime)
   end
 
   def map_mentions_data_to_chart(mentions_data)
-    mentions_data.each_with_object({}) do |m, hash|
-      date = m.date.strftime('%-d %b')
-      hash[date] = m.total_mentions
-    end
+    mentions_data.each_with_object({}){|(date, total_mentions), hash| hash[date.strftime('%-d %b')] = total_mentions }
```

Comment:
Looks like another formatting issue here - is linting enabled on this repo? Might save time in the long run!

```suggestion
    mentions_data.each_with_object({}) { |(date, total_mentions), hash| hash[date.strftime('%-d %b')] = total_mentions }
```